website/docs/r/hybrid_kubernetes_connected_cluster.html.markdown

```diff
@@ -0,0 +1,98 @@
+---
+subcategory: "HybridKubernetes"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_hybrid_kubernetes_connected_cluster"
+description: |-
+  Manages a HybridKubernetes connected cluster.
+---
+
+# azurerm_hybrid_kubernetes_connected_cluster
+
+Manages a HybridKubernetes connected cluster.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_hybrid_kubernetes_connected_cluster" "test" {
+  name                         = "example-connected-cluster"
+  resource_group_name          = azurerm_resource_group.test.name
+  agent_public_key_certificate = "xxxxxxxxxxxxxxxxxxx"
+  distribution                 = "gke"
+
+  identity {
+    type = "SystemAssigned"
+  }
+  infrastructure = "gcp"
+  location       = "West Europe"
+
+  tags = {
+    ENV = "Test"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this HybridKubernetes Connected Cluster. Changing this forces a new HybridKubernetes Connected Cluster to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the HybridKubernetes connected cluster should exist. Changing this forces a new HybridKubernetes connected cluster to be created.
+
+* `agent_public_key_certificate` - (Required) Base64 encoded public certificate used by the agent to do the initial handshake to the backend services in Azure. Changing this forces a new HybridKubernetes connected cluster to be created.
+
+* `identity` - (Required) An `identity` block as defined below.
+
+* `location` - (Required) The Azure Region where the HybridKubernetes connected cluster should exist. Changing this forces a new HybridKubernetes connected cluster to be created.
+
+* `distribution` - (Optional) The HybridKubernetes distribution running on this connected cluster. Changing this forces a new HybridKubernetes connected cluster to be created.
+
+* `infrastructure` - (Optional) The infrastructure on which the HybridKubernetes cluster represented by this connected cluster is running on. Changing this forces a new HybridKubernetes connected cluster to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the HybridKubernetes connected cluster.
+
+---
+
+An `identity` block exports the following:
+
+* `type` - (Required) Specifies the type of Managed Service Identity. Possible values are `SystemAssigned`.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the HybridKubernetes connected cluster.
+
+* `identity` - An `identity` block as defined below.
+
+* `provisioning_state` - The current deployment state of connected clusters.
```

Comment:
this field should be removed:

```suggestion
```

and the other computed fields should be added: